util/debug_switches.c

```diff
@@ -438,3 +444,11 @@ int debug_switch_test_delay_analyze_commit(void)
 {
     return debug_switches.test_delay_analyze_commit;
 }
+int debug_switch_all_incoherent(void)
+{
+    return debug_switches.replicant_latency;
```

Comment:
Maybe I'm misunderstanding, but I think there is a typo here...  Shouldn't this be 'all_incoherent'?

Reply:
YES .. thanks Joe .. !